javascript/javascript3/week1/apiresponse.js

```diff
@@ -0,0 +1,46 @@
+/* 
+2. Postal PIN code API http://www.postalpincode.in/Api-Details
+
+Postal PIN Code API responds the details of post offices by searching a specific postal pin code
+or a branch name within India. The responded json data shows the number of post offices, the type of post office
+whether it is "Main/Head" or a "Sub" post office and it's complete address means which city and state that
+post office belongs to. For example:
+
+a. GET https://api.postalpincode.in/pincode/{PINCODE}  ---- Which gives the number of postal offices and it's
+ addresses and related sub post offices based on the given pincode. The api responded json data is in array and
+ the data structure is "array of objects". Below is the json response data.
+
+
+[{"Message":"Number of pincode(s) found:3","Status":"Success","PostOffice":[{"Name":"Chaithanyapuri Colony",
```

Comment:
You could format the JSON by using a formatter or `JSON.stringify` to make it easier to read 🤓